src/components/characterInfo/CharacterInfo.js

```diff
@@ -0,0 +1,34 @@
+import LikeButton from '../likeButton/LikeButton';
+import './characterInfo.scss';
+
+function CharacterInfo() {
+    return (
+        <div className="page-content__character-info character-info">
+            <div className="character-info__poster">
+                <img className="character-info__image" src="content/img/wonder-woman.jpg" alt="" width="320" height="420"/>
+                <LikeButton class={'character-info'}></LikeButton>
```

Comment:
Когда просто текст, то скобки можно не писать
```suggestion
                <LikeButton class='character-info'></LikeButton>
```